subworkflows/local/reference_indexing.nf

```diff
@@ -7,6 +7,7 @@ include { REFERENCE_INDEXING_MULTI  } from '../../subworkflows/local/reference_i
 include { GUNZIP as GUNZIP_PMDBED   } from '../../modules/nf-core/gunzip/main.nf'
 include { GUNZIP as GUNZIP_PMDFASTA } from '../../modules/nf-core/gunzip/main.nf'
 include { GUNZIP as GUNZIP_SNPBED   } from '../../modules/nf-core/gunzip/main.nf'
+include { GUNZIP as GUNZIP_CM_FASTA } from '../../modules/nf-core/gunzip/main.nf'
```

Comment:
```suggestion
include { GUNZIP as GUNZIP_ELONGATED_FASTA } from '../../modules/nf-core/gunzip/main.nf'
```

Could you rename this module to make its connection to the respective parameter clearer?